client/overview/survey/icons/close.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { SVG, Path } from '@wordpress/primitives';
+
+export default (
+	<SVG
+		width="24"
+		height="24"
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 24 24"
+		aria-hidden="true"
+		focusable="false"
+	>
+		<Path d="M12 13.06l3.712 3.713 1.061-1.06L13.061 12l3.712-3.712-1.06-1.06L12 10.938 8.288 7.227l-1.061 1.06L10.939 12l-3.712 3.712 1.06 1.061L12 13.061z"></Path>
+	</SVG>
+);
```

Comment:
Surely we can use a standard woo or wordpress component for the close button? If we are proposing a new one, let's be clear about that and start the process of getting it merged upstream (WCPay component lib, Woo component lib, or WordPress core).

Reply:
indeed, we can try to use a standard component .